src/messages/javascript.js

```diff
@@ -152,7 +154,7 @@ function temporaryAPI(api) {
     code: apiToMessage(api),
     message: i18n._(`"${api}" can cause issues when loaded temporarily`),
     description: i18n._(oneLine`This API can cause issues when loaded
-      temporarily using about:debugging in Firefox unless you specify
+      temporarily using about:debugging in Thunderbird unless you specify
       applications|browser_specific_settings > gecko > id in the manifest.
```

Comment:
I know this is not part of your changeset, but "since we are here": 

`applications` is deprecated and we should use `browser_specific_settings` only. Maybe remove it here, so we do not have to deal with it later anymore?

Reply:
Sounds good, I'm just double checking to see if I can non-destructively modify the strings first.